deployments/rialto/OpenEthereum.Dockerfile

```diff
@@ -18,7 +18,8 @@ RUN update-ca-certificates && \
 
 ENV PATH="/root/.cargo/bin:${PATH}"
 ENV LAST_RUST_UPDATE="2020-08-17"
-RUN rustup update stable && \
+RUN rustup toolchain uninstall stable && \
```

Comment:
Without this change (and similar changes in other 2 files), I've been getting something like "failed to copy docs/html" when running docker-compose. Could rollback this fix, if it's only me

Reply:
FTR - this hasn't actually solved my problem && I still have to `rustup toolchain uninstall stable` first :/ But since I've see no problems with this on CI, I'm considering this as my local issue.